### PR TITLE
Added method to generate string with n spaces or n characters

### PR DESCRIPTION
**Pull request by oscargus**
Used in a number of places so makes sense to have a method.

### CODE REVIEW

**Comment by simonharrer** [comment]
LGTM

File: src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java [hunk 1]
```diff
@@ -669,4 +669,32 @@ public static String expandAuthorInitials(String name) {
...
+    }
+
+    /**
+     * Return a String with n copies of the char c
+     *
+     * @param n Number of copies
+     * @param c char to copy
+     * @return String with n copies of c
+     */
+    public static String nCharacters(int n, char c) {
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java]: Such a method already exists in Guava: http://docs.guava-libraries.googlecode.com/git-history/v10.0.1/javadoc/com/google/common/base/Strings.html#repeat%28java.lang.String,%20int%29
Should we use this method instead?
If not I would propose to change the name of `nCharacters` to `repeat(char, int)` and similarly `repeatSpaces(int)`

File: src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java [hunk 2]
```diff
@@ -669,4 +669,32 @@ public static String expandAuthorInitials(String name) {
...
+        StringBuilder resultSB = new StringBuilder();
```
Comment (tobiasdiez) [inline src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java]: Init StringBuilder with the right capacity `n`

File: src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java (hunk 1 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java]: I've seen references to it when googling, but wasn't sure which libraries we already use (there's also one in Apache StringUtils).

File: src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java (hunk 2 above)
Comment (oscargus) [inline src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java]: Good idea!

**Comment by tobiasdiez** [comment]
LGTM :+1:

**Pull request merged**